src/enum.js

```diff
@@ -5,7 +5,7 @@ module.exports = Enum;
 var ReflectionObject = require("./object");
 ((Enum.prototype = Object.create(ReflectionObject.prototype)).constructor = Enum).className = "Enum";
 
-var Namespace = require("./namespace"),
+var getNamespace = function () { return require("./namespace") },
```

Comment:
Isn't there another way, like reordering imports? Asking because there's already a more general [workaround in place](https://github.com/dcodeIO/protobuf.js/blob/master/src/namespace.js#L428) for similar cases that doesn't require calling a function each time, and instead sets these things up on instantiation.

Reply:
I see, neat! See update.

Ps. had some fun with `node --max_inlined_bytecode_size=0 test.js`, not sure if the extra function call would have mattered in execution, though using existing logic is better!